rectools/model_selection/cross_validate.py

```diff
@@ -36,6 +57,7 @@ def cross_validate(  # pylint: disable=too-many-locals
     ref_models: tp.Optional[tp.List[str]] = None,
     validate_ref_models: bool = False,
     on_unsupported_targets: ErrorBehaviour = "warn",
+    compute_timings: bool = False,
```

Comment:
Please add new argument to docstring

Reply:
do we really need this param? what's wrong if we always measure the time?